crates/subspace-farmer/src/lib.rs

```diff
@@ -23,6 +23,7 @@
 
 pub(crate) mod archiving;
 pub(crate) mod commitments;
+pub mod dsn;
```

Comment:
Does the whole module need to be public? We primarily exposed individual data structures before.